JSDemos/Demos/TreeList/MultipleRowSelection/description.md

```diff
@@ -1 +1,7 @@
-In&nbsp;this demo, the TreeList component uses the multiple selection mode. To&nbsp;select a&nbsp;row, click the check box located in&nbsp;this row. Note that selection is&nbsp;non-recursive by&nbsp;default, that&nbsp;is, only the clicked row is&nbsp;selected, but not its nested rows. To&nbsp;make it&nbsp;recursive, click the _&laquo;Recursive Selection&raquo;_ check box under the TreeList. You can also select all rows at&nbsp;once by&nbsp;clicking the check box in&nbsp;the column header.
\ No newline at end of file
+This demo enables the *"multiple"* **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode). This mode allows users to select multiple rows using checkboxes or keyboard shortcuts.
+
+The checkbox in the leftmost column header selects all rows. If you want to disable it, set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false**.
```

Comment:
>The checkbox in the leftmost column header selects all rows. If you want to disable **it**, set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false**.

I understand, "it" refers to "the checkbox" in your second sentence. However, this does not look quite clear.

```suggestion
The checkbox in the leftmost column header selects all rows. You can set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false** to disable the checkbox.
```